site-content/source/modules/ROOT/pages/resources.adoc

```diff
@@ -798,4 +798,42 @@ https://www.youtube.com/watch?v=td5QRYOeGag&list=PLU2OcwpQkYCy_awEe5xwlxGTk5UieA
 ----
 //end card
 
+//start card  
+[openblock,card resource-card text-center shadow relative level-Beginner]
+----
+[openblock,title mt-medium]
+------
+[discrete]
+=== Cassandra.Link
```

Comment:
something more descriptive please

Reply:
Acknowledged, I will make it more descriptive